<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>100</horstretch>
    <verstretch>100</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>1500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Grocery Store Manager 1.0</string>
  </property>
  <widget class="QWidget" name="widget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>569</width>
     <height>429</height>
    </size>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>550</height>
     </size>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
  background-color: #fbeee0;
  border: 2px solid #422800;
  border-radius: 30px;
  box-shadow: #422800 4px 4px 0 0;
  color: #422800;
  cursor: pointer;
  display: inline-block;
  font-weight: 600;
  font-size: 18px;
  padding: 0 18px;
  line-height: 50px;
  text-align: center;

}

QLineEdit{
  border: none;
  border: 2px solid #422800;
background-color: #fbeee0;
font-weight: 600;
  font-size: 18px;
}
QLabel{
font-weight: 600;
  font-size: 18px;
}
QWidget{

}</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="stackedWidgetPage1">
     <widget class="QLabel" name="MainLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>WholeSale Management System</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="UserName">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>149</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="UserLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>UserName:</string>
      </property>
     </widget>
     <widget class="QLabel" name="PassLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Password">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>162</y>
        <width>149</width>
        <height>25</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="Loginbtn">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>220</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Signupbtn">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>220</y>
        <width>101</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>SignUp</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="stackedWidgetPage2">
     <widget class="QLabel" name="AccountLabel">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Create New Account</string>
      </property>
     </widget>
     <widget class="QLabel" name="UserLabel_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>UserName:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Password_3">
      <property name="geometry">
       <rect>
        <x>178</x>
        <y>158</y>
        <width>149</width>
        <height>25</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="PassLabel_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLabel" name="PassLabel_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirm Pass:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Password_2">
      <property name="geometry">
       <rect>
        <x>178</x>
        <y>130</y>
        <width>149</width>
        <height>25</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="UserName_2">
      <property name="geometry">
       <rect>
        <x>178</x>
        <y>102</y>
        <width>149</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="createaccbtn">
      <property name="geometry">
       <rect>
        <x>192</x>
        <y>230</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>SignUp</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Backbtn_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>230</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="stackedWidgetPage3">
     <widget class="QPushButton" name="btnFindItem">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>40</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Supplier
Details</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnReturnItems">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>200</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Add
Supplier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAvailableItems">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Available
Items</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAddItem">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Add
 Item</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnReport">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>200</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Final
Report</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExpiredItems">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Customer
Details</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUpdateItem">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Update
Item</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSoldItems">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>120</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Add
Customer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSpoiledItem">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sell
Item</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
